Gemfile

```diff
@@ -44,7 +44,11 @@ group :development, :test do
   end
 
   # Run development and test tasks
-  gem 'rake', '~> 12.0'
+  if RUBY_VERSION >= '2.0.0'
+    gem 'rake', '~> 12.3'
+  else
+    gem 'rake', '~> 12.2.0'
+  end
```

Comment:
Sometimes, a fiddlier way of expressing the same thing avoids warnings output from Bundler. Like a ternary conditional operator, etc.

Reply:
I _have had_ warnings from Bundler telling me not to repeat myself. Perhaps they're _smarter_ now, perhaps that was an edge-case.